collection_php.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+// This line does nothing but create a connection
+$db = new PDO('mysql:host=db; dbname=collectionDB', 'root', 'password');
+
+// Set the fetch mode to assoc array
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+// This line flags an error with the code if something isn't connecting
+$db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+
+// Here we prepare the query, ready to run it
+$query = $db->prepare('SELECT * FROM `collection_table`;');
+
+// We actually run the query against the DB
+$query->execute();
+
+// Returns the full array from the database
+$result = $query->fetchAll();
+
+?>
+
+<!-- This form collects data to be uploaded to the database -->
+<h1>OverHyped Gaming</h1>
+<form method="post">
+    <pre>
+        <h3>Title</h3>
+        <input type="text" name="form_title" placeholder="new title" maxlength="20">
+    </pre>
+    <pre>
+        <h3>Date</h3>
+        <input type="date" name="form_release" placeholder="date">
+    </pre>
+    <pre>
+        <h3>Description</h3>
+        <input type="text" name="form_reason" placeholder="description" maxlength="600">
+    </pre>
+    <input type="submit">
+</form>
+
+
+<?php
+
+// Checks the fields have all been filled out
+if (isset($_POST['form_title']) && ($_POST['form_release']) && ($_POST['form_reason'])) {
```

Comment:
I found something that said it's better to use !empty instead of isset to make sure the forms are all complete. Don't know whether Ash would agree. I used it on mine and it seems to work fine. Just putting that out there.